packages/create-plugin/src/utils/utils.prettifyFiles.ts

```diff
@@ -1,16 +1,19 @@
 import { exec as nodeExec } from 'node:child_process';
 import { promisify } from 'node:util';
 import fs from 'node:fs';
+import { getPackageJson } from './utils.packagejson.js';
 
 const exec = promisify(nodeExec);
 
-export async function prettifyFiles(exportPath: string) {
+export async function prettifyFiles(exportPath: string, projectRoot = exportPath) {
   if (!fs.existsSync(exportPath)) {
     return '';
   }
 
+  const prettierVersion = getPrettierVersion() || '2';
```

Comment:
nit: if we're only ever calling this when prettier is used I'm not sure there's a need for the fallback. 🤔